src/Directory.Build.props

```diff
@@ -13,6 +13,8 @@
   </Target>
 
   <PropertyGroup>
+    <DebugType>Full</DebugType>
```

Comment:
Should this be conditioned on Configuration/Platform?

Reply:
I don't think so. 
Why would you want to turn this off for retail? (or have less info)

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/code-generation#debugtype

``` 
If you use Full, be aware that there's some impact on the speed and size of JIT optimized code and a small impact on code quality with full. We recommend pdbonly or no PDB for generating release code.
```

Reply:
> The following information applies only to compilers older than C# 6.0

Reply:
> For all compiler versions starting with C# 6.0, there is no difference between pdbonly and full.